src/components/Button/Button.js

```diff
@@ -41,9 +57,10 @@ Button.propTypes = {
 };
 
 Button.defaultProps = {
-  color: 'black',
-  width: '50px',
-  clickData: null
+  backgroundColor: null,
```

Comment:
So it's not required and u have given a default value which seems correct. But if the value is null what happens in line 18?